tests/libstorage/storageclass.go

```diff
@@ -348,3 +351,15 @@ func isLocalPV(pv k8sv1.PersistentVolume) bool {
 func isPVAvailable(pv k8sv1.PersistentVolume) bool {
 	return pv.Spec.ClaimRef == nil
 }
+
+func CreateBlockDataVolume(virtClient kubecli.KubevirtClient, namespace string) (*cdiv1.DataVolume, error) {
+	sc, foundSC := GetBlockStorageClass(k8sv1.ReadWriteOnce)
+	Expect(foundSC).To(BeTrue(), "Block storage is not present")
```

Comment:
So if you want to get rid of this (understandable if not) the "correct" way would be to use a label and have the test binary caller set it appropriately. Here's a reference:
https://github.com/kubevirt/kubevirt/pull/10124
This may or may not require changes in various testing scripts outside of kubevirt/kubevirt, such as for the purpose of downstream testing.